lib/coverband/utils/railtie.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Coverband
+  class Railtie < Rails::Railtie
+    # Coverband needs to be setup before any of the initializers
+    # to capture usage of things loaded by them
+    # if one uses before_eager_load as I did previously
+    # any files that get loaded as part of railties will have no coverage
+    config.before_initialize do
+      Coverband.configure
```

Comment:
This line currently fails if there is no coverband.rb. What do you think of changing this and just going with sensible defaults?

Reply:
Yeah I was thinking of trying to work towards the zero config side of things. I will do a pass and see if anything is really needed for basic running out of the gate. I am assuming for zero config we look for standard envs like REDIS_URL. We probably would want to give better errors if we can't detect anything.

Reply:
see latest commits, I think it would cover basic assumptions we could probably make it a bit more resilent but feels like a reasonable first pass.